package sxg;
message UserInfoBase{
    optional uint32 ver = 1; // version
    optional uint32 user_id = 2;
    optional uint32 user_name = 3;
    optional uint32 nick_name = 4;
    optional uint32 reg_time = 5;
    optional uint32 from = 6; // 1.wx 2.qq 3.mobile
    optional uint32 login_time = 7;
    optional uint32 last_login_time = 8;
    optional uint32 fresh_time = 9;
}
